Templates/QueueTrigger-Python/function.json

```diff
@@ -7,7 +7,7 @@
       },
       {
         "type": "table",
```

Comment:
VS uses 4 spaces for me for JSON. Not sure what;s the default, but we should all use the same...